### PR TITLE
RD-3353 Upgraded umzug to v3

### PR DESCRIPTION
**Pull request by kubama**
## Description
Upgraded umzug to v3

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
Proper unit test already exists - updated.

 

## Documentation
N/A

### CODE REVIEW

File: backend/migration.js
Comment (qooban) [inline backend/migration.js:96]: Cool refactor! Shorter and easier to read code 👍🏻

File: backend/migration.js
Comment (qooban) [inline backend/migration.js:44]: Is this comment still valid?

**Review by qooban — COMMENTED** [review]
Looks good! Added only minor comment.

Before final approval, let's discuss general things:
1. Did you test `cloudify-stage` and `cloudify-blueprint-composer` migrations? Any problems encountered there?
2. Let's discuss a plan for releasing new version. There is #76 and #95 . Should we wait for them and release new major version at once?

File: backend/migration.js
Comment (kubama) [reply backend/migration.js]: I would say it was useless even before the change - removed

**Comment by kubama** [comment]
> 1. Did you test `cloudify-stage` and `cloudify-blueprint-composer` migrations? Any problems encountered there?

Yes. It revealed some issues but I already managed to solve them. In the end the upgrade will be transparent - no updates on stage or composer side necessary.

> 2. Let's discuss a plan for releasing new version. There is #76 and #95 . Should we wait for them and release new major version at once?

Sounds like a plan. Is this going to be a breaking change so it requires a major release?

**Review by qooban — APPROVED** [review]
Let's discuss a plan for releasing new version. There is https://github.com/cloudify-cosmo/cloudify-ui-common/pull/76 and https://github.com/cloudify-cosmo/cloudify-ui-common/pull/95 . Should we wait for them and release new major version at once?
> Sounds like a plan. Is this going to be a breaking change so it requires a major release?

Reg. https://github.com/cloudify-cosmo/cloudify-ui-common/pull/76 - yeah, I suspect so, this one will break backward compatibility. I'll rebase it once you merge Umzug upgrade and try to prepare it for continuation

Reg. https://github.com/cloudify-cosmo/cloudify-ui-common/pull/95 - I hope it won't introduce breaking changes.

**Pull request merged**
